Sources/ResponsiveTextField/ResponsiveTextField.swift

```diff
@@ -368,6 +368,10 @@ extension ResponsiveTextField: UIViewRepresentable {
         uiView.text = text.wrappedValue
         uiView.textColor = textColor
         uiView.textAlignment = textAlignment
+        
+        DispatchQueue.main.async {
+            uiView.reloadInputViews()
+        }
```

Comment:
Is this dispatch necessary here?

Reply:
Hi @lukeredpath, we noticed it would not reload if it is not there, but we have also noticed a significant performance drop while testing this so I will close the PR for now until we have a better solution.